docs/repl.md

```diff
@@ -17,7 +17,35 @@ This would open a Chrome browser that you can control with the REPL interface. M
 wdio repl chrome -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY
 ```
 
-You can apply any available options for your REPL session. (See `wdio --help` for options.)
+```sh
+wdio repl android
+OR
```

Comment:
can we separate the code snippets similar to the example below?

Reply:
Fixed